cloudbackup_mon.sh

```diff
@@ -74,27 +68,35 @@ fi
 api_key=$1
 
 if [ -z "$2" ]; then
-	this_backup_conf_id=
+  this_backup_conf_id=
 else
-	this_backup_conf_id=$2
+  this_backup_conf_id=$2
 fi
 
 
 # Get username and agentid
 username=`cat /etc/driveclient/bootstrap.json | grep Username | awk '{print $3}' | sed -e 's/"//g' -e 's/,//g'`
 agentid=`cat /etc/driveclient/bootstrap.json | grep AgentId | awk '{print $3}' | sed -e 's/,//g'`
+ddi=`cat /etc/driveclient/bootstrap.json | grep AccountId | awk '{print $3}' | sed -e 's/"//g' -e 's/,//g'`
+api_hostname=`cat /etc/driveclient/bootstrap.json | grep ApiHostName | awk '{print $3}' | sed -e 's/"//g' -e 's/,//g'`
+
+
+token=`curl -s https://identity.api.rackspacecloud.com/v2.0/tokens  \
+    -X POST \
+     -d '{"auth":{"RAX-KSKEY:apiKeyCredentials":{"username":"'"$username"'","apiKey":"'"$api_key"'"}}}' \
+     -H "Content-type: application/json" | python -m json.tool | grep -A5 token | grep id | cut -d '"' -f4`
 
 # Get token
-token=`curl -s -I -H "X-Auth-Key: $api_key" -H "X-Auth-User: $username" https://auth.api.rackspacecloud.com/v1.0 | grep -vi 'vary:' | grep -i X-Auth-Token |awk {'print $2'}`
+#token=curl -s -I -H "X-Auth-Key: $api_key" -H "X-Auth-User: $username" https://auth.api.rackspacecloud.com/v1.0 | grep -vi 'vary:' | grep -i X-Auth-Token |awk {'print $2'}
```

Comment:
We should either leave this command untouched (including backticks ``` `` ```) or remove the comment entirely.